test/scenarios/rhel-9.2-microshift-source-from-latest-ec.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Planning to move the file to `scenarios-periodics` after https://github.com/openshift/microshift/pull/2149/ merges.
@dhellmann @pacevedom I guess we'll run all periodics in a single job?

Reply:
Yes, that's what I had in mind. I think @pacevedom 's work will make that easy.